### ISSUE TITLE
Add option to make Threema Web more compact

### ISSUE THREAD

**Issue by ovalseven8** [threema-ch/threema-web#495]
When I use Threema Web on my desktop with a 24" monitor, everything is fine more or less. However, when I am on my laptop (13.3") Threema Web needs to much space. Usually, I scale to 90 % then.

What would be a cool:
Add a "compact mode" for smaller devices so that e.g. the contact list needs less space. On the 13.3" monitor I can see about 4 contacts in the list because every contact/item needs too much space.

All in all, it's about making Threema Web more responsive. I mean, it's responsive but there could be many optimizations.

**Comment by dbrgn** [comment]
Yes, this could be implemented using regular responsive design (CSS media queries).

**Comment by lgrahl** [comment]
/me is wondering why this doesn't have more upvotes.

**Comment by rugk** [comment]
…because we have big screens. :wink:
